default.py

```diff
@@ -39,7 +40,10 @@ def build_url(query):
     xbmcplugin.addDirectoryItem(handle=addon_handle, url=url, listitem=li, totalItems=2)
 
     # Get a list of the games from Lutris
-    args = settings.getSetting('lutris_executable') + ' --list-games --json'
+    if settings.getSetting('custom_path') in [True, 'True', 'true', 1]:
```

Comment:
So just drop the `in [True, 'True', 'true', 1]`? I tried to keep the style in `settings.getSetting('installed') in [True, 'True', 'true', 1]:` in line 47. I liked the idea of of checking the output of `get.Setting` against a list of possible values. It might be a bit redundant though.